src/Resources/templates/trait.php.twig

```diff
@@ -12,7 +12,7 @@ use {{ use_statement.name}} as {{ use_statement.alias }};
  * This is the guy that the main class needs to require, and ensures that everything is glued
  * together
  */
-trait {{ class_name }}Traits
+trait {{ class_name }}Trait
```

Comment:
Stefan included `hostnet/phpcs-tool`, and one of it's rules is that traits are suffixed with `Trait`, not `Traits`. This is, indeed, a big bc break.